lms/templates/emails/enroll_email_allowedmessage.txt

```diff
@@ -19,10 +19,10 @@ ${_("To access the course visit {course_about_url} and register for the course."
 % else:
 
 ${_("To access the course, you will first need to register for Residential MITx at {site_name}"
-    "with your Kerberos ID or Touchstone account."
-    "If you do not have a Kerberos ID and need to set up a Touchstone account,"
-    "please contact your instructor for detailed directions.\n").format(
-        site_name=site_name
+        "with your Kerberos ID or Touchstone account."
+        "If you do not have a Kerberos ID and need to set up a Touchstone account,"
+        "please contact your instructor for detailed directions.\n").format(
+            site_name=site_name
```

Comment:
sorry, not understand. what is the actual difference in these two statements above ?

Reply:
in PR #84 they had only removed the tab space, so in order to revert it, I added it back.